src/storage/src/region.rs

```diff
@@ -103,6 +103,10 @@ impl<S: LogStore> Region for RegionImpl<S> {
     async fn alter(&self, request: AlterRequest) -> Result<()> {
         self.inner.alter(request).await
     }
+
+    async fn close(&self) -> Result<()> {
```

Comment:
Looks like this PR contains changes in #987 😕

Reply:
sorry @evenyag  a little bit confused by the #987 as it's the number of the current PR.
Do you mean I've resolved conflicts with issues?

Reply:
Sorry, I means #970

Reply:
Ah, ok.
That came as a result of the rebase. 
I find it strange that it's not showing as a conflict. 
@evenyag  what would you recommend here?

Reply:
Could you please try [this](https://stackoverflow.com/questions/16306012/github-pull-request-showing-commits-that-are-already-in-target-branch)?

> Use the Edit button when viewing the PR in GitHub to change the base branch to something other than master. Then switch it back to master and it will now correctly show only the changes from the most recent commits.

Reply:
ooh.. thanks @evenyag  for "googling that for me" 😅.. Didn't even thought that it might be that simple.
Will put it in my "TIL" notes.